styleguide/components/00-settings/_vars.scss

```diff
@@ -4,10 +4,10 @@ $uuid: unique-id();
 
 // Border radius.
 $radius-0: 0 !default;
-$radius-1: 8px !default;
-$radius-2: 10px !default;
-$radius-3: 18px !default;
-$radius-4: 30px !default;
+$radius-1: .5rem !default;
+$radius-2: .625rem !default;
```

Comment:
Do we need .5 and .625?
Can you spot the difference? 
https://codepen.io/mordicus/pen/XEWydM 

(Will radius-0 ever be not '0' ?)

Reply:
No, but this might be handy when overriding a radius somewhere in the code. 

The values of all radiuses are not final yet in the design, so I just kept some of the old ones so the code still compiles.

Reply:
Alright, let's make sure we don't allow two pixel variations again.
I'm sure we can work together with design on these things and point them out early.